### PR TITLE
feat(ui): Hover stacktrace preview

### PR DESCRIPTION
**Pull request by matejminar**
Kapture 2020-08-26 at 10 22 58

This is a feature-flagged experimental proof of concept.

### CODE REVIEW

**Comment by BYK** [comment]
@matejminar this looks pretty cool, still active?

**Comment by matejminar** [comment]
We still want to explore this idea, however, it has a low priority.
Closing now and will reopen once we are ready to spend a few hours on this again.

File: src/sentry/static/sentry/app/components/stacktracePreview.tsx [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    event.preventDefault();
+  };
+
+  render() {
+    const {organization, children} = this.props;
+    const {event} = this.state;
+    const exception = event?.entries.find(e => e.type === 'exception')?.data;
+    const stacktrace = exception?.values[0]?.stacktrace;
+
+    if (!organization.features.includes('stacktrace-hover-preview')) {
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/components/stacktracePreview.tsx:51]: should we maybe verify the feature flag before calling this component?

**Review by priscilawebdev — APPROVED** [review]
left 1 suggestion but it looks good to me ✅

File: src/sentry/static/sentry/app/components/stacktracePreview.tsx (hunk 1 above)
Comment (matejminar) [reply src/sentry/static/sentry/app/components/stacktracePreview.tsx:51]: I was also considering it but with the way things are structured I'd rather do it like this 👍

**Pull request merged**